scripts/ocm-local-up-for-ingress.sh

```diff
@@ -41,19 +41,42 @@ nodes:
     hostPort: 9443
     protocol: TCP
 EOF
+
+# Wait for hub cluster nodes to be ready
+echo "Waiting for hub cluster nodes to be ready..."
+kubectl --context "${hubctx}" wait --for=condition=ready nodes --all --timeout=300s
+
 echo "Installing an nginx ingress controller into the hub cluster..."
-kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/refs/tags/controller-v1.12.1/deploy/static/provider/kind/deploy.yaml
+kubectl --context "${hubctx}" apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/refs/tags/controller-v1.12.1/deploy/static/provider/kind/deploy.yaml
```

Comment:
Why add the explicit context specification here? We statically know that the current context is the desired context.